### PR TITLE
global api securisation

### PR DESCRIPTION
**Pull request by MathildeJNL**

### CODE REVIEW

File: src/main/java/com/templateproject/api/ApiApplication.java [hunk 1]
```diff
@@ -1,15 +1,36 @@
...
 	public static void main(String[] args) {
 
 		SpringApplication.run(ApiApplication.class, args);
 	}
 
+	@Bean
+	public CommandLineRunner run(
+			RoleRepository roleRepository
+	) {
+
```
Comment (KevinRoig) [inline src/main/java/com/templateproject/api/ApiApplication.java]: Line to delete

**Review by KevinRoig — APPROVED** [review]
Ok. All comments are here to help us because Securiity is hard to understand without it.

File: src/main/java/com/templateproject/api/controller/ServiceController.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    public Service updateService(Principal principal, @PathVariable UUID serviceId, @RequestBody Service serviceToModify){
+        Service serviceInBdd = this.serviceRepo
+                .findById(serviceId)
+                .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND, "Le service n'a pas été trouvé."));
+
+        User petsitter = this.userRepo
+                .findByEmail(principal.getName())
+                .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND, "Votre utilisateur n'a pas été trouvé."));
+
+        if(petsitter.getServices().contains(serviceInBdd)) {
```
Comment (bastienapp) [inline src/main/java/com/templateproject/api/controller/ServiceController.java]: il manque un espace après le if: if (petsitter ...

Un coup de CTRL + MAJ + ALT + L :smile:

**Review by bastienapp — APPROVED** [review]
Je valide.

Merci de corriger un espace avant de fusionner

**Pull request merged**